src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,11 @@
+.stopwatch {
+  position: relative;
+  justify-content: center;
+  align-items: center;
+  display: flex;
+  @include circle ($vertical-circle-size, $horizontal-circle-size);
+  border: $border-circle-pin;
```

Comment:
It's recommended to use a variable name that more accurately describes its purpose, such as `$border-width`.